packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -662,9 +662,11 @@ abstract class StatelessWidget extends Widget {
 
   /// Describes the part of the user interface represented by this widget.
   ///
-  /// The framework calls this method when this widget is inserted into the
-  /// tree in a given [BuildContext] and when the dependencies of this widget
-  /// change (e.g., an [InheritedWidget] referenced by this widget changes).
+  /// The framework calls this method when this widget is inserted into the tree
+  /// in a given [BuildContext] and when the dependencies of this widget change
+  /// (e.g., an [InheritedWidget] referenced by this widget changes). This
+  /// method can potentially be called in every frames, and you should keep in
+  /// mind that it should not have any side effects beyond building a widget.
```

Comment:
per style guide: avoid empty phrases:

```suggestion
  /// method can potentially be called in every frames and should not have any 
  /// side effects beyond building a widget.
```